src/components/common/Select.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+import DropdownArrow from 'components/common/DropdownArrow';
+
+const StyledSelect = styled.select`
+  flex: 1;
+  border-width: 0;
+  margin: 0.1em 0;
+  padding: 0.5em 0.7em;
+  appearance: none;
+  height: 40px;
+  font-size: 0.9em;
+  ${({ theme }) => `
+    background-color: ${theme.background.input};
+    border-radius: ${theme.border.radius};
+  `}
+`;
```

Comment:
I'm thinking how can we split this into two files, because for large components with complex css this will get really out of hand. 
What do you think about splitting your component and your styled declaration into two files? Not sure how would I name them though, just throwing the question to see what we can come up with.